<launch>
	<param name="use_sim_time" value="true"/>
	
	<node pkg="road_recognizer" type="intensity_partition" name="intensity_partition" output="screen"/>

	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>
	<include file="$(find velodyne_height_map)/launch/amsl_velodyne_heightmap.launch"/>
	<!-- <include file="$(find velodyne_height_map)/launch/velodyne_heightmap_nodelet.launch"/> -->
	
	<node pkg="rosbag" name="player" type="play" args="--clock /home/amsl/bagfiles/cats/2019/left_start.bag"/>
	<node pkg="rviz" name="rviz" type="rviz" args="-d /home/amsl/.rviz/otsu.rviz"/>
</launch>
